### PR TITLE
Add version logs

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add version logs

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -30,6 +30,16 @@
 __version__ = "0.1.2"
 __title__ = "si-cloc"
 
+from thoth.analyzer import __version__ as __analyzer__version__
+from thoth.common import __version__ as __common__version__
+
+__service_version__ = (
```
Comment (fridex) [inline app.py:36]: `__component_version__`?

**Review by fridex — APPROVED** [review]

File: app.py (hunk 1 above)
Comment (pacospace) [reply app.py]: Updated in 9cbf1055558ba38bf4c15d4838059163041a5ff3

**Pull request merged**